{
  "artifacts": [],
  "command": "main",
  "experiment": {
    "base_dir": "/home/deep/nico/t3p/baselines/script",
    "dependencies": [
      "baselines==0.1.5",
      "gym==0.9.6",
      "numpy==1.14.1",
      "rllab==0.1.0",
      "sacred==0.7.4",
      "tensorflow==1.5.0"
    ],
    "mainfile": "run_rllab_pois_sacred.py",
    "name": "POIS",
    "repositories": [
      {
        "commit": "2c2d368fef9988d0133bcde0c0dfe74d5d8c8452",
        "dirty": false,
        "url": "https://github.com/T3p/baselines.git"
      }
    ],
    "sources": [
      [
        "run_rllab_pois_sacred.py",
        "_sources/run_rllab_pois_sacred_da4bbb548b3110fd0119e82c0a4c09b9.py"
      ]
    ]
  },
  "fail_trace": [
    "Traceback (most recent call last):\n",
    "  File \"/home/deep/.local/lib/python3.5/site-packages/tensorflow/python/framework/op_def_library.py\", line 510, in _apply_op_helper\n    preferred_dtype=default_dtype)\n",
    "  File \"/home/deep/.local/lib/python3.5/site-packages/tensorflow/python/framework/ops.py\", line 1022, in internal_convert_to_tensor\n    ret = conversion_func(value, dtype=dtype, name=name, as_ref=as_ref)\n",
    "  File \"/home/deep/.local/lib/python3.5/site-packages/tensorflow/python/framework/ops.py\", line 866, in _TensorTensorConversionFunction\n    (dtype.name, t.dtype.name, str(t)))\n",
    "ValueError: Tensor conversion requested dtype float64 for Tensor with dtype float32: 'Tensor(\"one_hot:0\", shape=(100, 100), dtype=float32)'\n",
    "\nDuring handling of the above exception, another exception occurred:\n\n",
    "Traceback (most recent call last):\n",
    "  File \"/home/deep/.local/lib/python3.5/site-packages/sacred/config/captured_function.py\", line 46, in captured_function\n    result = wrapped(*args, **kwargs)\n",
    "  File \"script/run_rllab_pois_sacred.py\", line 156, in main\n    positive_return=positive_return)\n",
    "  File \"script/run_rllab_pois_sacred.py\", line 130, in train\n    clipping=clipping, entropy=entropy, positive_return=positive_return)\n",
    "  File \"/home/deep/nico/t3p/baselines/baselines/pois/pois.py\", line 478, in learn\n    clustered_target_pdf = tf.matmul(tf.reshape(target_pdf_episode, (1, -1)), episode_clustering_matrix)[0][:max_index]\n",
    "  File \"/home/deep/.local/lib/python3.5/site-packages/tensorflow/python/ops/math_ops.py\", line 2022, in matmul\n    a, b, transpose_a=transpose_a, transpose_b=transpose_b, name=name)\n",
    "  File \"/home/deep/.local/lib/python3.5/site-packages/tensorflow/python/ops/gen_math_ops.py\", line 2516, in _mat_mul\n    name=name)\n",
    "  File \"/home/deep/.local/lib/python3.5/site-packages/tensorflow/python/framework/op_def_library.py\", line 546, in _apply_op_helper\n    inferred_from[input_arg.type_attr]))\n",
    "TypeError: Input 'b' of 'MatMul' Op has type float32 that does not match type float64 of argument 'a'.\n"
  ],
  "heartbeat": "2018-10-03T18:53:08.337263",
  "host": {
    "ENV": {},
    "cpu": "Intel(R) Xeon(R) CPU E5-4610 v2 @ 2.30GHz",
    "hostname": "bellman",
    "os": [
      "Linux",
      "Linux-4.4.0-134-generic-x86_64-with-Ubuntu-16.04-xenial"
    ],
    "python_version": "3.5.2"
  },
  "meta": {
    "command": "main",
    "options": {
      "--beat_interval": null,
      "--capture": null,
      "--comment": null,
      "--debug": false,
      "--enforce_clean": true,
      "--file_storage": null,
      "--force": false,
      "--help": false,
      "--loglevel": null,
      "--mongo_db": null,
      "--name": null,
      "--pdb": false,
      "--print_config": false,
      "--priority": null,
      "--queue": false,
      "--sql": null,
      "--tiny_db": null,
      "--unobserved": false,
      "COMMAND": null,
      "UPDATE": [
        "env=cartpole",
        "policy=linear",
        "iw_method=rbis",
        "logdir=cp_linear",
        "bound=max-d2",
        "delta=0.4",
        "gamma=0.99",
        "center=True",
        "njobs=5",
        "seed=160"
      ],
      "help": false,
      "with": true
    }
  },
  "resources": [],
  "result": null,
  "start_time": "2018-10-03T18:53:04.898395",
  "status": "FAILED",
  "stop_time": "2018-10-03T18:53:08.342260"
}